test/test-ipc.c

```diff
@@ -247,37 +248,39 @@ static void on_read_listen_after_bound_twice(uv_stream_t* handle,
 
   fprintf(stderr, "got %d bytes\n", (int)nread);
 
-  ASSERT(uv_pipe_pending_count(pipe) > 0);
+  ASSERT_GT(uv_pipe_pending_count(pipe), 0);
   pending = uv_pipe_pending_type(pipe);
-  ASSERT(nread > 0 && buf->base && pending != UV_UNKNOWN_HANDLE);
+  ASSERT_GT(nread, 0);
+  ASSERT(buf->base);
```

Comment:
FWIW, Node has `ASSERT_NULL` and `ASSERT_NOT_NULL` macros. I suppose `ASSERT_PTR_EQ` and `ASSERT_PTR_NE` macros might come in handy too.

Reply:
Yes, I thought about it. I'll add them then.